src/opentelemetry/launcher/configuration.py

```diff
@@ -139,6 +158,26 @@ def configure_opentelemetry(
             satellite. Defaults to `False`.
     """
 
+    log_levels = {
+        "NOTSET": NOTSET,
+        "DEBUG": DEBUG,
+        "INFO": INFO,
+        "WARNING": WARNING,
+        "ERROR": ERROR,
+        "CRITICAL": CRITICAL,
+    }
+
+    if match(r"(?i){}".format("|".join(log_levels.keys())), log_level) is None:
```

Comment:
this could be simplified by using `log_level.upper() in log_levels`

Reply:
Fixed